### PR TITLE
Fix triple double-quote in docstring

### PR DESCRIPTION
**Pull request by ddurieux**

### CODE REVIEW

**Comment by Seb-Solon** [comment]
Same here. Any rule for that?

**Comment by ddurieux** [comment]
perhaps with: pep257 --select=D300 alignak/*

I will try

**Comment by ddurieux** [comment]
I have added a test ;)

File: alignak/property.py [hunk 1]
```diff
@@ -61,9 +61,9 @@
 from alignak.util import to_float, to_split, to_char, to_int, unique_value, list_split
 import logging
 
-__all__ = ['UnusedProp', 'BoolProp', 'IntegerProp', 'FloatProp',
+__all__ = ('UnusedProp', 'BoolProp', 'IntegerProp', 'FloatProp',
            'CharProp', 'StringProp', 'ListProp',
-           'FULL_STATUS', 'CHECK_RESULT']
+           'FULL_STATUS', 'CHECK_RESULT')
```
Comment (Seb-Solon) [inline alignak/property.py]: Not related to the PR

Comment (ddurieux) [inline alignak/property.py]: Yes but pep257 have error on this

**Pull request merged**